scripts/manage-flaky-tests.sh

```
@@ -0,0 +1,328 @@
+#   remove    Re-enable a test (removes from quarantine.json)
+#   list      Print all currently quarantined tests
+#   validate  Check quarantine.json is structurally valid
+#   report    Print a Markdown table (useful for PR descriptions)
+#
+
+set -euo pipefail
+
+# Paths
+REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+REGISTRY_PATH="${REPO_ROOT}/backend/src/test/flaky/quarantine.json"
+SCHEMA_PATH="${REPO_ROOT}/backend/src/test/flaky/quarantine.schema.json"
```

Shellcheck (0.11.0)

[warning] SCHEMA_PATH appears unused. Verify use (or export if used externally).

(SC2034)